sympy/integrals/tests/test_integrals.py

```diff
@@ -1731,3 +1731,9 @@ def test_issue_21034():
 
     assert integrate(f2, x) == \
         log(x**2 + S(1)/12)/2 + 2*sqrt(3)*cosh(cos(4))*atan(2*sqrt(3)*x)
+
+
+def test_issue_4187():
+    x = Symbol('x')
```

Comment:
The symbol x is already defined at the top of the file so it does not need to be redefined here.

Reply:
sorry defined it by mistake , removed it now.